### PR TITLE
Switch on async-timeout > 4

### PR DESCRIPTION
**Pull request by shadchin**
## What do these changes do?

Add support new async-timeout and switch on them.

## Are there changes in behavior for the user?

No

## Related issue number

#247
#268

## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -7,7 +7,9 @@ module = "aiojobs"
 author = "Andrew Svetlov"
 author-email = "andrew.svetlov@gmail.com"
 home-page = "https://github.com/aio-libs/aiojobs"
-requires=[]
+requires=[
+    "async-timeout >= 4"
```
Comment (asvetlov) [inline pyproject.toml:11]:
```suggestion
    "async-timeout >= 4.0.0"
```
Let's keep the full version

**Review by asvetlov — APPROVED** [review]

**Comment by asvetlov** [comment]
The PR is good.
New aiojobs release requires switching to Github Actions CI

I cannot estimate when I can find a time for this migration, and help is welcome!

**Comment by ddc67cd** [comment]
Could you make a tag with this version, please? There's a problem using aiojobs with aiohttp because of different versions of async_timeout

**Comment by asvetlov** [comment]
I'd like to merge #277 first

**Pull request merged**